types/office-runtime/index.d.ts

```diff
@@ -18,10 +19,101 @@ Copyright (c) Microsoft Corporation
  * Office runtime namespace.
  */
 declare namespace OfficeRuntime {
+    /**
+     * Encapsulates methods that interacts with the Office UI
+     * @beta
+     */
+    interface UI {
+        /**
+         * Get the ribbon of the Office application.
+         */
+        getRibbon(): Promise<Ribbon>;
+    }
+    /**
+     * Encapsulates all the functionality provided to manage the state of the Office ribbon.
+     * @beta
+     */
+    interface Ribbon {
+        /**
+         * Stages the updates to be made to the ribbon.
+         * Note that this API only requests an update. The actual UI update to the ribbon is controlled by the
+         * Office application and hence the exact timing of the ribbon update (or refresh) cannot be determined
+         * by the completion of this API.
+         * @param input Represents the updates to be made to the ribbon. Note that the only changes made to the ribbon
+         * are those specified in the input parameter. In all other respects the ribbon is not changed.
+         */
+        requestUpdate(input: RibbonUpdaterData): Promise<void>;
+    }
+    /**
+     * Represents the parts of the ribbon that the call of requestUpdate will change.
+     * @beta
+     */
+    interface RibbonUpdaterData {
+        /**
+         * Array of the tabs whose state is set with call of requestUpdate.
```

Comment:
Nit: "with call of" -> "with a call of"

Also, should it be "call of" or "call to"?